file-formats/cota/type/zif_aff_cota_v1.intf.abap

```diff
@@ -9,14 +9,27 @@ INTERFACE zif_aff_cota_v1
     "! <p class="shorttext">Communication Type</p>
     "! Communication type
     BEGIN OF co_comm_type,
-      "! <p class="shorttext">RFC</p>
-      "! RFC
-      rfc  TYPE ty_comm_type VALUE 'R',
       "! <p class="shorttext">HTTP</p>
       "! HTTP
       http TYPE ty_comm_type VALUE 'H',
     END OF co_comm_type.
 
+  "! <p class="shorttext">Multitenancy Mode</p>
+  "! $values {@link zif_aff_cota_v1.data:co_multimode_type}
+  TYPES ty_multi_type TYPE c LENGTH 30.
+
+  CONSTANTS:
+    "! <p class="shorttext">Multitenancy Mode</p>
+    "! Multitenancy mode
+    BEGIN OF co_multimode_type,
+      "! <p class="shorttext">Cross Client</p>
+      "! Cross client
+      cross_client       TYPE ty_multi_type VALUE 'C',
+      "! <p class="shorttext">Client Specific</p>
+      "! Client dependent
+      client_dependent TYPE ty_multi_type VALUE 'D',
```

Comment:
I am not an expert, but are the multitenancy modes really "crossClient" and "clientDependent"? I cannot explain it, but it sounds somehow strange to me. But I might be wrong.

Reply:
Cross Client and Client Specific is the correct terminus I found out couple of weeks ago. Therefore I changed the text.
The variable is still called "client dependent", that is not perfect, but fine for me. Is it acceptable?

Reply:
I think it is pretty close. Therefore it would be acceptable for me.

However, if "Client Specific" is the better term it should be easy to change it, shouldn't it?

Reply:
^^ This would have the advantage the correct terminus is also shown in the JSON file

Reply:
done :)